tests/headers/derive-partialeq-union.hpp

```diff
@@ -0,0 +1,7 @@
+// bindgen-flags: --with-derive-partialeq --impl-partialeq
+
+// Deriving PartialEq for rust unions is not supported.
+union ShouldNotDerivePartialEq {
```

Comment:
If we make this comment use `///` or `/** ... */` style, then it will appear in the expectations too, which is sometimes useful for checking that the trait you expect to derive or not is as expected.